### PR TITLE
docker 17.06.0

### PR DESCRIPTION
**Pull request by mkjaer**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by mkjaer** [comment]
I asked the bump-formula-pr tool to change the url to 'https://github.com/docker/docker-ce.git'
I'm unsure if this will work or if I manually have to change the urls.

File: Formula/docker.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Docker < Formula
   desc "Pack, ship and run any application as a lightweight container"
   homepage "https://www.docker.com/"
   url "https://github.com/docker/docker.git",
-      :tag => "v17.05.0-ce",
-      :revision => "89658bed64c2a8fe05a978e5b87dbec409d57a0f"
+      :tag => "v17.06.0-ce",
```
Comment (fxcoudert) [inline Formula/docker.rb]:
```Cloning into '/Users/brew/Library/Caches/Homebrew/docker--git'...
warning: Could not find remote branch v17.06.0-ce to clone.
```

Comment (ilovezfs) [inline Formula/docker.rb]: Right, the URL needs to be https://github.com/docker/docker-ce.git as he mentioned above

**Comment by ilovezfs** [comment]
The install method will need a tweak as well since the build is failing
```
==> hack/make.sh dynbinary-client
Failed to execute: hack/make.sh
```

Also, the versioned alias needs to be bumped from

`Aliases/docker@17.05 -> ../Formula/docker.rb`
to
`Aliases/docker@17.06 -> ../Formula/docker.rb`

**Comment by ilovezfs** [comment]
@JCount did a bit of research here and it looks like the formula is going to require a substantial rewrite, or we'll have to remove it from homebrew/core entirely, deprecating in favor of the Cask.

So I'm closing this PR but thanks nonetheless for opening it @mkjaer

**Pull request closed without merging**